controllers/products.go

```diff
@@ -126,3 +126,43 @@ func deleteProduct(productID *uuid.UUID, tx *sql.Tx) error {
 
 	return nil
 }
+
+func GetProduct(productID *uuid.UUID) (*models.ProductData, error) {
+	tx, err := mysqldb.DBConnector.ConnectSystem()
+	if err != nil {
+		return nil, err
+	}
+
+	product, err := mysqldb.Functions.GetProductByID(*productID, tx)
```

Comment:
Why don't you use *uuid. UUID parameter in the Functions.GetProductByID method?

Reply:
Good question. :) I fix it.